content/blog/2021-08-11-getting-funding.md

```diff
@@ -0,0 +1,80 @@
+fellowship](https://boards.greenhouse.io/tweag/jobs/4638654002) offer a great
+way to get interested students, or with tweag even more professional coders
+into the project, test their potential (e.g. for a later masters/PhD) and get
+new contributors for a project. However, the support is for a rather short time
+(12 weeks).  In Germany, the DFG sometimes puts out calls for research software
+(e.g. [2016 call for sustainable research software
+development](https://www.dfg.de/foerderung/info_wissenschaft/2016/info_wissenschaft_16_71/)
+or [2019 call for quality management in research
+software](https://www.dfg.de/foerderung/info_wissenschaft/2019/info_wissenschaft_19_44/index.html))
+
+
+### Part B: How can we get more funding for scientific software?
+
+Overall, we got the impression, that many software projects are funded by being
+hidden within larger grant proposals. While this works to get some funding, it
+leads to an undesired invisibility of research software development, which
+makes it difficult to get funding for projects. On the other hand decision
+makers seem to be aware, that software is necessary, and there are even calls
+which explicitly ask for details on what software funding there will be. As
+such, it seems important to get grant-writers to stop the practice of hiding or
+obfuscating software development in their grants but to explicitly list them
+in the requirements and/or goals of the grant.  To achieve this it would also
+be important to convince PIs that proper funding for research software
+development is important, and present the advantages of having permanent
+research software engineers.  Those include (but are not limited to):
+
+- A RSE stays, and thus the knowledge about the code remains in house for future reference
+- A RSE has a higher incentive to produce higher quality code, that stays useable even if they leave the institution
+- Code development can be sped up if a RSE serves multiple groups by not having
+  to replicate the same code multiple times (in contrast to individual
+  PhDs/PostDocs who only code for "themselves")
+
+A further argument, which needs some more work could be the time saved by
+employing RSEs. This would require a survey of PIs on how much time they, or
+their group spend on code development, and an estimate of how much time could
+be saved by employing specialists for some of the tasks. This data would also
+be important for higher level decision makers in order to show the value of
+RSEs in research. To this end, it would be good to obtain more and clearer
+statistics to demonstrate use and visibility as mentioned above. For some
+languages, download statistics can easily be obtained by package managers (e.g.
+conda/pip for python):
+- [Pip Download badge for git](https://img.shields.io/pypi/dm/numpy?label=downloads%20%7C%20pip&logo=PyPI)
+- [Conda download badge for git](https://img.shields.io/conda/dn/conda-forge/numpy?label=downloads%20%7C%20conda&logo=Conda-Forge
```

Comment:
There is a missing closing parenthesis here for the link

Reply:
Thanks!